keps/sig-auth/4317-pod-certificates/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: PodIdentity certificates, PodCertificate volumes, and in-cluster kube-apiserver client certificates
```

Comment:
Is there a shorter, snappier title that we can use? (if so, change the issue title first, then reflect that change here)

Reply:
Shortened to "Pod Certificates"